CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.7
+
+_Nov 4, 2021_
+
+- 👁 Move virtualization logic to hook (#3079) @m4theushw
+- 🌍 Add hebrew localization (he-IL) (#3028) @ColdAtNight
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.0.0-beta.7` / `@mui/x-data-grid-pro@v5.0.0-beta.7`
+
+#### Breaking changes
+
+- [core] Prefix selectors from `useGridContainerProps` with `unsafe` (#3002) @flaviendelangle
+
+  We are temporarily making the dimension APIs of the grid private to be able to clean them in a future minor release. The current approach is causing a lot of useless re-renders.
+  Please reach to us if you are really using any of those selectors to help us understand what are the community needs around this subject.
+
+  The following selectors were prefixed by `unstable_`. Use the provided alternatives.
```

Comment:
@flaviendelangle I'm not sure to get the point. The alternative is to use `unstable_gridContainerSizesSelector`?

Reply:
```suggestion
  The following selectors were prefixed with `unstable_`. Use the provided alternatives.
```

Side note: we have "unsafe" and "unstable" in the code base. Is "unstable" the correct one to use?

Reply:
From what I understand, `unsafe_` is for methods that have never been documented, and should not be used by the developers. Whereas `unstable_` will be available back after the refactoring, or at least method providing similar information.

Reply:
It should be `unstable_` everywhere, it's my bad :/

Reply:
Do you want to correct it before release?

Reply:
OK I'll do a quick PR